Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -89,7 +109,7 @@ private Vector3 GetCameraControlTranslation()
         deltaPosition += InputCurve(Input.GetAxis("Horizontal")) * this.transform.right;
         deltaPosition += InputCurve(Input.GetAxis("Vertical")) * this.transform.forward;
```

Comment:
I think you need the same control mode checks for the "Vertical" axis. Testing movement in the ManualCameraControl scene, the arrow keys work correctly, but WASD controls behave as they used to. I believe they should both respect the control mode.

I also noticed that Shift is the hard coded key binding for fast/slow, which makes sense because it matches the unity editor, but we've recently changed the default mouse control to Shift as well. We probably need to make the fast/slow controls configurable or take away Shift for mouse control even though it's the best option given the unsaved scene + waSd dilemma. I'm not sure, but what we have now is confusing and by defaulting to Shift movement with mouse control is always fast. Maybe I should make an issue for this outside of your change?